test/unit/player.test.js

```diff
@@ -824,3 +824,19 @@ expect(3);
   player.language('en-GB');
   strictEqual(player.localize('Good'), 'Brilliant', 'Ignored case');
 });
+
+test('should return correct values for canPlayType', function(){
+  var tag = TestHelpers.makeTag();
+  var container = document.createElement('div');
+  var fixture = document.getElementById('qunit-fixture');
+
+  container.appendChild(tag);
+  fixture.appendChild(container);
+
+  var player = new Player(tag, { techOrder: ['techFaker'] });
+
+  ok(player.canPlayType('video/mp4') === 'maybe', 'player can play mp4 files');
```

Comment:
This should be changes to
`equal(player.canPlayType('video/mp4'), 'maybe', 'player can play mp4 files')`. And below.